src/pages/video-test.tsx

```diff
@@ -48,19 +48,20 @@ const EggheadPlayer: React.FC<{videoResource: VideoResource}> = ({
   )
 
   return (
-    <div className="-mx-5 -mt-5">
+    <div className="video-test -mx-5">
       {videoResource.hls_url && (
         <div
           ref={playerContainer}
           className="relative grid grid-cols-1 lg:grid-cols-12 "
         >
-          <div className="relative z-10 lg:col-span-9">
+          <div className="relative z-10 lg:col-span-9 pb-[4.5rem]">
             <Player
               muted
               autoplay
               crossOrigin="anonymous"
               className="font-sans"
               poster={videoResource.poster}
+              // className="cueplayer-react-fullscreen"
```

Comment:
uncomment it and switch to fullscreen to get styles applied

Reply:
this can be removed
```suggestion
```